spec/swagger.yaml

```diff
@@ -489,9 +616,16 @@ definitions:
       caffeine:
         type: integer
         description: mg of caffeine per 100 ml/mg/unit
+      alcohol:
+        type: integer
+        description: "volume percent (without decimal seperator)"
+        example: 450 # means 4,5 vol. %
       price:
         type: integer
         default: 150
+      barcode:
```

Comment:
A product can have multiple barcodes, currently. It would be nice to preserve this.

Reply:
If you can wait a few days, the current endpoints under `/barcodes` in Mete will be in v1.

Reply:
Yes, sure!

Reply:
This might be interesting: https://space-market.github.io/API/preview/v1/#tag/barcodes